readme.md

```diff
@@ -98,6 +98,28 @@ Tasks
 This plugin adds a `printVersion` task, which will echo the project's configured version
 to standard-out.
 
+
+☑️ Configuration cache
+---
+External process calls to `git` in this plugin are compatible with Gradle's [Configuration Cache](https://docs.gradle.org/current/userguide/configuration_cache.html).
+
+Since we're using CC compatible APIs, we leverage Gradle's ability to reuse outputs of `git` commands upon successive invocations.
```

Comment:
```suggestion
Since we're using Configuration Cache compatible APIs, we leverage Gradle's ability to reuse outputs of `git` commands upon successive invocations.
```